cmd/ci-operator-configresolver/main.go

```diff
@@ -5,6 +5,7 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	"github.com/openshift/ci-tools/pkg/api"
```

Comment:
...
Im going to look into that linter.

Reply:
I don't have the motivation to fix it :(